toast-tk-web-agent/src/main/java/io/toast/tk/agent/ui/panels/FileChoosePanel.java

```diff
@@ -0,0 +1,121 @@
+package io.toast.tk.agent.ui.panels;
+
+import io.toast.tk.agent.config.AgentConfigProvider;
+import io.toast.tk.agent.ui.i18n.UIMessages;
+import io.toast.tk.agent.ui.utils.ConfigTesterHelper;
+import io.toast.tk.agent.ui.utils.PanelHelper;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import javax.swing.*;
+import javax.swing.border.TitledBorder;
+import java.awt.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.io.IOException;
+import java.util.Properties;
+
+
+public class FileChoosePanel extends AbstractPanel {
+
+    private static final Logger LOG = LogManager.getLogger(FileChoosePanel.class);
+    private final String label;
+    private final EnumError errorType;
+    private JButton fileSearch;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove the "fileSearch" field and declare it as a local variable in the relevant methods. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1450)